### PR TITLE
Adding Mission Statement + Reworking the intro 

### PR DESCRIPTION
**Pull request by spier**
In a call between @maxcapraro @lenucksi and @MaineC we thought it would be good to have a clearly defined mission statement, so that anybody contributing to this repo knows what we are working towards.

What I did in this PR:

* added a first mission statement
* reworked the intro with the goal to make it easier to read, and focus on "contributions are welcome" more.

### CODE REVIEW

File: README.md
Comment (MaineC) [inline README.md]:
```suggestion
This repository contains the InnerSource Patterns collected by the [innersourcecommons.org](InnerSource Commons). These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies][isc-getting-started]. 
```

**Comment by MaineC** [comment]
Thank you for writing down what we discussed in the call this morning. I added one change suggestion, other than that LGTM.

File: README.md
Comment (maxcapraro) [reply README.md]:
```suggestion
This repository contains the InnerSource Patterns collected by the [innersourcecommons.org](InnerSource Commons). These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource][isc-getting-started]
```
(This is @MaineC 's suggestion - minus the word methodologies)

File: README.md
Comment (maxcapraro) [inline README.md]:
```suggestion
Our mission in this working group
- Collect and document agreed upon best practices of how to do InnerSource - in the form of patterns
- Continuously publish the most mature patterns as an ebook
```

File: README.md
Comment (maxcapraro) [inline README.md]:
```suggestion
```

I'd drop it as it doesn't add specific information - so That way we can keep it lean?

**Review by maxcapraro — CHANGES_REQUESTED** [review]
Excellent @spier :) I am excited!

File: README.md
Comment (spier) [reply README.md]: Sounds good.

File: README.md
Comment (spier) [reply README.md]: You suggestion won't create a valid link I think. But I understand what you are after and will fix it accordingly.

Comment (spier) [reply README.md]: Please see the latest version of this branch/PR.

File: README.md
Comment (maxcapraro) [inline README.md]: (In general all changes look good - not sure if I am overly picky with this comment):

I always feel a lot depends on "first sentences" because they are a door opener.

Would you feel the following is a improvement or a regression?

```suggestion
This repository contains the InnerSource Patterns collected by the [InnerSource Commons][isc-website]. These patterns are InnerSource best practices codified in a specific format to make it easy to understand, evaluate, and reuse them. 
```

**Review by maxcapraro — APPROVED** [review]
General approval; one additional thought about first paragraph.

File: README.md
Comment (spier) [reply README.md]: Looks good to me, feel free to merge it in. However then you will also have to remove the link the reference `[isc-getting-started]` as you are not using that link anymore.

File: README.md
Comment (maxcapraro) [inline README.md]:
```suggestion
```

**Comment by maxcapraro** [comment]
So. Do we merge or do we wait until #141 is merged and we also have the "moral mandate"?

**Comment by spier** [comment]
Let's wait for #141 to be approved.

**Review by rrrutledge — APPROVED** [review]

**Comment by lenucksi** [comment]
#141 has just been merged.
No objections against these edits have been voiced so far, two approvals are here. I think this is nicely written and hence will merge it.
If anyone wants a change to this please reach out to us.

**Comment by spier** [comment]
Nice! Stuff is happening :)

**Pull request merged**
